tests/proximal_gradient_test.py

```diff
@@ -42,6 +42,8 @@ def stepsize_schedule(t: int) -> float:
   return stepsize_schedule
 
 
+n_calls = 0
```

Comment:
same comment as in the other file: uppercase global variables and docstring for tests

Reply:
done